modules/components/analytics/web/org.wso2.carbon.pc.analytics.web/src/web/js/actions.js

```diff
@@ -160,7 +203,7 @@ function drawAvgExecuteTimeVsProcessVersionResult(renderElement) {
                         });
                     }
                     render(renderElementID, dataset, 'AVG Execution Time (ms)', 'Process Version');
-                }else{
+                } else {
                     //Error message
```

Comment:
If there's nothing to add in the else scope, then we can remove the else section. No point having an empty scope.